src/Sql/Sql/Auditing/Model/ServerAuditModel.cs

```diff
@@ -19,6 +19,34 @@
 
 namespace Microsoft.Azure.Commands.Sql.Auditing.Model
 {
+    public enum AuditStateType { Enabled, Disabled };
+
+    public enum AuditActionGroups
+    {
+        BATCH_STARTED_GROUP,
+        BATCH_COMPLETED_GROUP,
+        APPLICATION_ROLE_CHANGE_PASSWORD_GROUP,
+        BACKUP_RESTORE_GROUP,
+        DATABASE_LOGOUT_GROUP,
+        DATABASE_OBJECT_CHANGE_GROUP,
+        DATABASE_OBJECT_OWNERSHIP_CHANGE_GROUP,
+        DATABASE_OBJECT_PERMISSION_CHANGE_GROUP,
+        DATABASE_OPERATION_GROUP,
+        DATABASE_PERMISSION_CHANGE_GROUP,
+        DATABASE_PRINCIPAL_CHANGE_GROUP,
+        DATABASE_PRINCIPAL_IMPERSONATION_GROUP,
+        DATABASE_ROLE_MEMBER_CHANGE_GROUP,
+        FAILED_DATABASE_AUTHENTICATION_GROUP,
+        SCHEMA_OBJECT_ACCESS_GROUP,
+        SCHEMA_OBJECT_CHANGE_GROUP,
+        SCHEMA_OBJECT_OWNERSHIP_CHANGE_GROUP,
+        SCHEMA_OBJECT_PERMISSION_CHANGE_GROUP,
+        SUCCESSFUL_DATABASE_AUTHENTICATION_GROUP,
+        USER_CHANGE_PASSWORD_GROUP
+    }
+
+    public enum StorageKeyKind { Primary, Secondary };
+
```

Comment:
It seems that these variable is not referenced. Maybe you can add this modification in other commit.

Reply:
@wyunchi-ms 
They are referenced in unmodified files, like SetAzSqlDatabaseAudit.cs.